pom.xml

```diff
@@ -1047,7 +1047,7 @@
 		<openmrs.version>${project.version}</openmrs.version>
 
 		<springVersion>4.1.4.RELEASE</springVersion>
-		<hibernateVersion>4.3.9.Final</hibernateVersion>
+		<hibernateVersion>4.3.10.Final</hibernateVersion>
```

Comment:
are you updating to the latest hibernate version,and if so is it really this??

Reply:
@HerbertYiga you may look through [this](https://talk.openmrs.org/t/there-seems-to-be-a-backward-incompatibility-in-hibernate-versions/23401/18)